jenkins/jk-setup.sh

```diff
@@ -133,12 +133,11 @@ elif [[ $COMPILER == *icc* ]]; then
   export CC=`which icc`
   export CXX=`which icc`
   export FC=`which ifort`
-  export ExtraCMakeOptions="${ExtraCMakeOptions} -Dvc=OFF"
```

Comment:
I think, this line here was for the icc compiler. Is now VC working well with all versions of icc?